broccoli/transforms/transform-babel-plugins.js

```diff
@@ -0,0 +1,16 @@
+const Babel = require('broccoli-babel-transpiler');
+
+module.exports = function(tree) {
+  let options = {
+    sourceMaps: true,
+    plugins: [
+      ['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true, legacy: false }],
+      ['@babel/plugin-proposal-class-properties'],
+    ],
+    parserOpts: {
+      plugins: [['classProperties', {}], ['decorators', { decoratorsBeforeExport: true }]],
+    },
```

Comment:
can you check if this works fine without the `parserOpts`? I vaguely remember that this was only needed for those babel steps that did not use the corresponding plugins

Reply:
it does! most excellent!